src/actions/merged.ts

```diff
@@ -28,16 +28,16 @@ export async function merged(
         configuration.reset_label !== undefined &&
         labelNames.indexOf(configuration.reset_label) !== -1
     ) {
-        balance = 0
+        // Force pointsReceived up enough to make balance default
+        pointsReceived = 0 - oldBalance
```

Comment:
`-oldBalance`?

Reply:
sure, was trying to decide if it should default to 0 or 5 because I thought 5 was the default for some reason